Cargo.toml

```diff
@@ -22,13 +23,15 @@ regex = "1.11.1"
 sha1 = "0.10.6"
 base64 = "0.22.1"
 defer-heavy = "0.1.0"
+sync-ptr = "0.1.2"
 #GZIP/ZLIB in pure rust
 libflate = "2.1.0"
 
 
 ## SSL
 rustls = { version = "0.23.23", optional = true }
 rust-tls-duplex-stream = { version = "0.1.1", optional = true }
+serde = { version = "1.0.218", features = ["derive"] }
```

Comment:
this should only be in dev-dependency? probably just need to add the derive deature to the dev dep